scout/adapter/mongo/case.py

```diff
@@ -852,6 +880,7 @@ def load_case(self, config_data, update=False, keep_actions=True):
             old_evaluated_variants = list(
                 self.evaluated_variants(case_obj["_id"], case_obj["owner"])
             )
+            self.update_case_data_sharing(old_case=existing_case, new_case=case_obj)
```

Comment:
This is the only change in this function. I don't know what's going on with black